ids/_helpers.py

```diff
@@ -17,14 +22,14 @@ def dearmour(armoured: str) -> str:
 
 from cryptography.hazmat.primitives import serialization
 from cryptography.hazmat.primitives.asymmetric import ec, rsa
-def parse_key(key: str):
+def parse_key(key: str) -> PublicKeyTypes | PrivateKeyTypes:
```

Comment:
so, I *had* typing here already, but for some reason someone got this error:
```
ImportError: cannot import name 'PrivateKeyTypes' from 'cryptography.hazmat.primitives.asymmetric.types' 
```

I'm not really sure what to do about that, and everything else worked for them, so I just removed the type hints.